conda/core/initialize.py

```diff
@@ -746,7 +747,10 @@ def make_entry_point(target_path, conda_prefix, module, func):
         # no shebang needed on windows
         new_ep_content = ""
     else:
-        new_ep_content = "#!%s\n" % join(conda_prefix, get_python_short_path())
+        from .portability import generate_shebang_for_entry_point
```

Comment:
Can we move this to the top of the file or does this trigger a circular import?

Reply:
Let's see :)